bazel/linux/qemu.bzl

```diff
@@ -51,11 +51,12 @@ fi
 test -z "$KERNEL" || QEMU_FLAGS+=("-kernel" "$KERNEL")
 test -z "$INTERACTIVE" || KERNEL_FLAGS+=("init=/bin/sh")
 
-QEMU_FLAGS+=("-append" "${{KERNEL_FLAGS[*]}}")
+QEMU_FLAGS+=("-append" "${{KERNEL_FLAGS[*]}} ${{KERNEL_OPTS[*]}}")
+QEMU_FLAGS+=("${{EMULATOR_OPTS[@]}}")
 
 echo Running qemu: "$QEMU_BINARY" "${{QEMU_FLAGS[@]}}"
 if [ -z "$INTERACTIVE" ]; then
-    "$QEMU_BINARY" "${{QEMU_FLAGS[@]}}" | tee "$OUTPUTFILE"
```

Comment:
Does this work for you?

I have to change it to:
```
"$QEMU_BINARY" "${{QEMU_FLAGS[@]}}" < /dev/null | tee "$OUTPUT_FILE"
```

With strace, qemu is stuck at: `ioctl(0</dev/pts/1>, SNDCTL_TMR_START or TCSETS, {B38400 opost -isig -icanon -echo ...}`

With gdb, qemu receives a signal `Thread 1 "qemu-system-x86" received signal SIGTTOU, Stopped (tty output).`